Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -1441,64 +1134,125 @@ script:
       description: The ID of the agent that was shutdown.
       type: String
   - arguments:
-    - default: false
-      description: 'A free-text search term, will match applicable attributes (sub-string match). Note: A device''s physical addresses will only be matched if they start with the search term  (not if they contain the search term).'
-      isArray: false
+    - description: "A free-text search term, will match applicable attributes (sub-string\
+        \ match). Note: A device's physical addresses will only be matched if they\
+        \ start with the search term  (not if they contain the search term)."
       name: query
-      required: false
-      secret: false
-    - default: false
-      description: A comma-separated list of agents IDs to shutdown.
-      isArray: false
+    - description: A comma-separated list of agents IDs to shutdown.
       name: agent_id
-      required: false
-      secret: false
-    - default: false
-      description: The ID of the network group.
-      isArray: false
+    - description: The ID of the network group.
       name: group_id
-      required: false
-      secret: false
-    deprecated: false
     description: Sends an uninstall command to all agents that match the input filter.
-    execution: false
     name: sentinelone-uninstall-agent
-  dockerimage: demisto/python3:3.10.1.26972
+  - arguments:
+    - auto: PREDEFINED
+      defaultValue: 'true'
+      description: Access global list? (Same as `tenant` flag in API docs)
+      name: global
+      predefined:
+      - 'true'
+      - 'false'
+    - description: Comma separated list of group IDs to filter by
+      name: group_ids
+    - description: Comma separated list of siteIDs to filter by
+      name: site_ids
+    - description: Comma separated list of account IDs to filter by
+      name: account_ids
+    - defaultValue: '0'
+      description: How many records to skip (for paging)
+      name: skip
+    - defaultValue: '1000'
+      description: Max number of records to return
+      name: limit
+    description: Add a hash to the blocklist ("blacklist" in SentinelOne documentation).
+      If the `global` flag is `true`, then group_ids, site_ids, and account_ids are
+      ignored.
+    name: sentinelone-get-blocklist
+  - arguments:
+    - description: SHA1 Hash to Add to Global Blocklist
+      name: sha1
+    - defaultValue: XSOAR
+      description: String describing the source of the block
+      name: source
+    - auto: PREDEFINED
+      name: os_type
+      predefined:
+      - windows
+      - linux
+      - macos
+      required: true
+    - defaultValue: Blocked from XSOAR
+      description: Note stored in SentinelOne about the block
+      name: description
+    description: Add a hash to the Global blocklist in SentinelOne
+    name: sentinelone-add-hash-to-blocklist
+    outputs:
+    - contextPath: SentinelOne.AddHashToBlocklist
+      description: Dictionary of information about block status
```

Comment:
You need to add here a complete context path and not only to the root path.e.g.:  ```SentinelOne.AddHashToBlocklist.id``` (this is just an example)